Fix #10128 and Automate generation of SVGs in explorations

donosco99 opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes #10128  
2. This PR does the following: made the proccess of generating SVGs for explorations from admin tab easier 

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The linter/Karma presubmit checks have passed locally on your machine.
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
  - This lets reviewers restart your CircleCI tests for you.
- [x] The PR is made from a branch that's **not** called "develop".

## PR Pointers

- Oppiabot will notify you when you don't add a PR_CHANGELOG label. If you are unable to do so, please @-mention a code owner (who will be in the Reviewers list), or ask on [Gitter](https://gitter.im/oppia/oppia-chat).
- For what code owners will expect, see the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
- Make sure your PR follows conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays
- Never force push. If you do, your PR will be closed.

donosco99 committed:
automate svg generation for explorations.

donosco99 committed:
Merge branch 'develop' into exp-svg-gen

donosco99 committed:
added backend tests

donosco99 commented:
![auto_gen](https://user-images.githubusercontent.com/31371571/89366806-369fdf00-d6f5-11ea-8f77-3cab711e6fcb.gif)

seanlip commented on core/controllers/admin.py:
```diff
@@ -604,12 +604,23 @@ class ExplorationsLatexSvgHandler(base.BaseHandler):
+            self.render_json({
+                'number_of_explorations_left_to_update': '%d' % (
+                    number_of_explorations_left_to_update)
+            })
```
else raise Error. Always close off the if/else loops fully.

donosco99 replied:
done

seanlip commented:
@nithusha21 @DubeySandeep PTAL as codeowners!

donosco99 commented:
unassigning codeowners as a little more optimization is required

donosco99 committed:
Made the number of explorations in a batch configurable.

donosco99 committed:
Merge branch 'develop' into exp-svg-gen

donosco99 committed:
minor fixes

seanlip commented:
@donosco98 -- tests are failing, PTAL!

donosco99 committed:
fix failing test

seanlip commented:
@donosco98 sorry -- could you please also merge #9919 into your branch? Otherwise this branch doesn't have the recent migration changes and it's hard to test.

donosco99 committed:
increase coverage

donosco99 committed:
Merge branch 'develop' into exp-svg-gen

donosco99 committed:
reuse images for duplicate raw_latex values

donosco99 committed:
increase the default value of number of svgs in a batch

seanlip reviewed (approved):
This passed a test run. LGTM (and congrats)!

donosco99 committed:
Merge branch 'develop' into exp-svg-gen

donosco99 committed:
fix failing tests

donosco99 committed:
Merge branch 'develop' into exp-svg-gen

donosco99 committed:
better naming

donosco99 committed:
adress review comments of #9985

donosco99 commented:
I've also addressed the deferred changes from the PR #9985 .
@DubeySandeep , @aks681 , @nithusha21 PTAL as codeowners

nithusha21 reviewed (approved):
LGTM for codeowner files!

seanlip commented:
@DubeySandeep @nithusha21, one note -- could you please explicitly confirm that this addresses the issues in https://github.com/oppia/oppia/issues/10128 before this PR is merged? Thanks!

DubeySandeep reviewed (approved):
LGTM for the codeowner files! Also, it fixes all the comments of #10128.

donosco99 committed:
added a todo to track upgrade to mathjax 3

donosco99 committed:
Merge branch 'develop' into exp-svg-gen

donosco99 commented:
> could you please explicitly confirm that this addresses the issues in #10128

I've added a todo for upgrade to Mathjax 3 and now this PR addresses all the issues in #10128.

donosco99 commented:
@aks681 , @kevintab95 PTAL as code-owners. thanks

nithusha21 reviewed (approved):
Thanks for pointing that out Sean. Yes, the if statements and the loop has been removed in the test file, so this does look good to me.

aks681 reviewed (approved):
Lgtm as codeowner

kevintab95 reviewed (approved):
LGTM for codeowner file. Thanks @donosco98!
